packages/overlay/src/integrations/sentry/components/performance/TransactionDetail.tsx

```diff
@@ -131,19 +140,27 @@ export default function TransactionDetail({ showAll }: { showAll: boolean }) {
             <tbody>
               {transactionsList.map(txn => (
                 <tr key={txn.event_id} className="hover:bg-primary-900">
-                  <td className="text-primary-200 w-3/5 truncate whitespace-nowrap px-6 py-4 text-left text-sm font-medium">
+                  <td className="text-primary-200 w-2/5 truncate whitespace-nowrap px-6 py-4 text-left text-sm font-medium">
                     {/* Ref: https://developer.mozilla.org/en-US/docs/Web/API/Window/btoa */}
                     <Link
                       className="truncate hover:underline"
                       to={`/explore/traces/${txn.contexts?.trace?.trace_id}/spans/${txn.contexts?.trace?.span_id}`}
                     >
-                      {txn.event_id}
+                      {txn.event_id.substring(0, 8)}
```

Comment:
This `.substring(0, 8)` thing should become a utility function so we don't have to repeat the same logic and magic number (`8`) everywhere.